neuralop/layers/codano.py

```diff
@@ -0,0 +1,607 @@
+from functools import partial
+import logging
+import numpy as np
+import torch
+import torch
+from torch import nn
+import torch.nn.functional as F
+from .fno_block import FNOBlocks
+from .spectral_convolution import SpectralConv1d, SpectralConv2d, SpectralConv3d
+
+
+# Codomain Attention Blocks
+class CODABlocks(nn.Module):
```

Comment:
I'm confused by this structure. Is the CODABlock a model, or a layer? If not a model, what else is required to turn this block into a full CODA-NO? I would also suggest naming the layer file `coda_blocks` or similar to disambiguate between the block and the model, if a difference exists.

Reply:
Sounds good